packages/opencensus-web-instrumentation-zone/src/interaction-tracker.ts

```diff
@@ -33,7 +33,12 @@ export type AsyncTask = Task & {
 };
 
 export class InteractionTracker {
+  // Allows to track several events triggered by the same user interaction in the right Zone.
+  private shouldCreateNewZone = true;
```

Comment:
Rather than keeping a separate boolean for this, what would you think about just making the current event tracing zone be optional and resetting it to `undefined` in the timer?

So something like `private currentEventTracingZone?: Zone`